src/arg_max_min.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+import {Tensor, sizeOfShape} from './lib/tensor.js';
+import {reduceMax, reduceMin, selectValuesToReduce} from './reduce.js';
+import {squeeze} from './squeeze.js';
+
+/**
+ * Get the index location of the minimum or maxmium values of all the input values along the axes.
+ * @param {Tensor} input
+ * @param {Function} reduceFunc
+ * @param {MLArgMinMaxOptions} [options]
+ * @return {Tensor}
+ */
+export function argMaxMin(
+    input,
+    reduceFunc,
+    {
+      axes = null,
+      keepDimensions = false,
+      selectLastIndex = false,
+    } = {}) {
+  const inpAxes = axes ?? new Array(input.rank).fill(0).map((_, i) => i);
```

Comment:
Might be worth adding a comment explaining the default axes by pointing to the spec.

Reply:
Done.